packages/salesforcedx-vscode-core/src/sfdxProject/sfdxProjectConfig.ts

```diff
@@ -75,8 +74,8 @@ export default class SfdxProjectConfig {
     return SfdxProjectConfig.instance;
   }
 
-  public static async getValue(key: string): Promise<AnyJson | undefined> {
+  public static async getValue<T extends JsonArray | string | undefined>(key: string): Promise<T> {
```

Comment:
@jeffb-sfdc narrowing to JsonArray is not appropriate. The config contains maps, arrays strings.